harvard-lts/CURIOSity
# Adding Beta Viewer link below the Mirador Viewer.

**Adding Beta Viewer link below the Mirador Viewer.**
* * *

**JIRA Ticket**: [LTSVIEWER-263](https://jira.huit.harvard.edu/browse/LTSVIEWER-263)

# What does this Pull Request do?
This adds a new link "Try new beta viewer" below the Mirador viewer on the Item detail pages.

# How should this be tested?

A description of what steps someone could take to:
* Pull in the `LTSVIEWER-263` branches of both mps-embed and mps-viewer and have them running locally first!
* Add new `VIEWER_URL` variable to your .env file. Value is: `https://localhost:23017`
* Build the CURIOSity container off of the `LTSVIEWER-263` branch
* Go to any item in any collection.
* Notice the new "Try new beta viewer" link next to "Open full screen viewer" link below the Mirador Viewer.
* Clicking on the link should take you to your local mps-viewer with the item in the new Mirador 3.0 viewer.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@f8f8ff @enriquediaz
